app/src/components/About.vue

```diff
@@ -0,0 +1,89 @@
+    class="h-screen fixed w-full"
+    style="z-index: 10000; background-color: #ffffffdd"
+  >
+    <div class="px-4 md:px-12 m-auto bg-white">
+      <div class="border-grey-500 border-2 px-4 md:px-12 relative">
+        <div class="absolute top-6 right-6">
+          <!-- ToDo. Remove classes when #131 is merged. We'd have the 'icon' utility class. -->
+          <XIcon
+            @click="$emit('toggleAbout')"
+            class="h-10 w-10 md:h-14 md:w-14 object-contain stroke-2 text-grey-400 stroke-current cursor-pointer"
+          />
+        </div>
+
+        <div class="mt-8">
+          <h1 class="font-medium"><span class="text-teal">d</span>Grants</h1>
+        </div>
+
+        <!-- ToDo. build version -->
+        <div class="mt-2">build 0.0.0.0</div>
+
+        <div class="mt-16">
+          <span>Dezentralize Gitcoin – Workstream Team: </span>
+          <span :key="contributor.login" v-for="(contributor, index) in $options.contributors">
+            <span
+              ><a href="{{contributor.profile}}" target="_blank">{{ contributor.name }}</a></span
+            >
+            <span v-if="index !== $options.contributors.length - 1">, </span><span v-else>.</span>
+          </span>
+        </div>
+
+        <div class="mt-16">
+          <!-- ToDo. Tweaks classes when #131 is merged. -->
+          <!-- ToDo. Link to a test grant (since dgrant would be a grant) -->
+          <router-link
+            to="#"
+            class="
+              items-center
+              gap-2
+              whitespace-nowrap
+              uppercase
+              font-medium
+              bg-gray-500
+              text-white
+              px-8
+              py-4
+              hover:bg-gray-400
+            "
+          >
+            <span>donate to <span class="text-teal">d</span>grants</span>
+          </router-link>
+        </div>
+
+        <div class="mt-16 mb-8 block md:flex gap-x-8">
+          <div>
+            <a href="https://github.com/dcgtc/dgrants/blob/main/LICENSE" target="_blank" rel="noreferrer noopener"
+              >Licensed under AGPL-3.0 License</a
+            >
+          </div>
+          <div><a href="https://github.com/dcgtc/dgrants" target="_blank" rel="noreferrer noopener">Github</a></div>
+          <div class="ml-auto">
+            <a href="https://support.gitcoin.co" target="_blank" rel="noreferrer noopener">Help & Support</a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { CloseIcon as XIcon } from '@fusion-icons/vue/interface';
+// ToDo. Connect & Configure all-contributor app
+import contributorsData from '../../../.all-contributorsrc.json';
+
+export default defineComponent({
+  name: 'About',
+  props: {
+    showAbout: Boolean,
+  },
+  components: {
+    XIcon,
+  },
+  contributors: contributorsData.contributors,
+  emits: ['toggleAbout'],
+});
+</script>
```

Comment:
wouldn't this a view instead of a component ?

Reply:
I think views are meant for "routed pages", right? Since this is going to be a modal loading on top of any page, I think it should be a component. But I might be wrong!

Reply:
I'd agree that this should be a component 👍

Reply:
> I think views are meant for "routed pages", right? Since this is going to be a modal loading on top of any page, I think it should be a component.
Agreed/correct!